tests/test_codegen_directive.cpp

```
@@ -110,6 +110,63 @@ TEST(NativeFnSigs, DirectivesRecorded) {
+    Program prog = parser.parseProgram();
+
+    // Verify the directive parameter was parsed
+    ASSERT_FALSE(prog.empty());
+    auto *fn = std::get_if<std::unique_ptr<FnStmt>>(&prog[0]);
+    ASSERT_NE(fn, nullptr);
+    ASSERT_FALSE((*fn)->directives.empty());
+    auto &d = (*fn)->directives[0];
+    EXPECT_EQ(d.name, "native");
+    ASSERT_EQ(d.params.size(), 1u);
+    EXPECT_EQ(d.params[0].key, "");
+    EXPECT_EQ(d.params[0].value, "base64");
```

Cppcheck (2.20.0)

[style] The function 'peek' is never used.

(unusedFunction)